src/applications/lgy/coe/containers/introduction-content/COEStatuses/COEAvailable.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+const COEAvailable = () => (
+  <>
+    <va-alert status="info">
+      <h3 slot="headline">You already have a COE</h3>
```

Comment:
I think this needs to be an h2. Preceding header is an H1.

Reply:
Derp, updated